src/pyclts/transcriptionsystem.py

```diff
@@ -110,6 +114,43 @@ def _update_regex(self):
             map(re.escape, sorted(self.sounds, key=lambda x: (len(x),
                 -ord(x[0])), reverse=True))))
 
+    def _update_diacritic_regex(self):
+        """ Initializes the regexes for diacritics from the diacritic dictionnary.
+
+        The structure of self.diacritic_regexes is:
+
+        ```
+            {type: (regex_pre, regex_post)}
+        ```
+
+
+        """
+        self._diacritic_regexes = {}
+
+        for d_type in self.diacritics:
+            pre_diacritics = []
+            post_diacritics = []
+
+            for d in self.diacritics[d_type]:
+                if d is not None: # Syllabic has None as a value: is this intended ?
```

Comment:
I noticed a strange row in CLTS's bipa diacritics, which leads to a diacritic "syllabic" with grapheme `None`. Is this expected ? 

https://github.com/cldf-clts/clts/blob/cccee296b1e54e653e1b4bea103bf0e870072765/pkg/transcriptionsystems/bipa/diacritics.tsv#L63

If this is a bug then this line `if d is not None` is not necessary.

Reply:
The bug is in the data. https://github.com/cldf-clts/clts/issues/129 
Thanks for spotting, the line can be deleted.